src/pages/login/CertificateGenerate.tsx

```diff
@@ -28,191 +21,49 @@ const CertificateGenerate: FC = () => {
     return <Navigate to="/ui" replace={true} />;
   }
 
-  const closeModal = () => {
-    setModalOpen(false);
-  };
-
-  const openModal = () => {
-    setModalOpen(true);
-  };
-
-  const createCert = (password: string) => {
-    closeModal();
-    setGenerating(true);
-
-    const worker = new Worker(
-      new URL("../../util/certificate?worker", import.meta.url),
-      { type: "module" },
-    );
-
-    worker.onmessage = (event: MessageEvent<Certs>) => {
-      setCerts(event.data);
-      setGenerating(false);
-      worker.terminate();
-    };
-
-    worker.onerror = (error) => {
-      console.error("Web Worker error:", error);
-      setGenerating(false);
-      worker.terminate();
-    };
-
-    worker.postMessage(password);
-  };
-
-  const downloadBase64 = (name: string, base64: string) => {
-    const linkSource = `data:application/octet-stream;base64,${base64}`;
-    const downloadLink = document.createElement("a");
-
-    downloadLink.href = linkSource;
-    downloadLink.download = name;
-    downloadLink.click();
-  };
-
-  const downloadText = (name: string, text: string) => {
-    const data = encodeURIComponent(text);
-    const linkSource = `data:text/plain;charset=utf-8,${data}`;
-    const downloadLink = document.createElement("a");
-
-    downloadLink.href = linkSource;
-    downloadLink.download = name;
-    downloadLink.click();
-  };
-
-  const crtFileName = `lxd-ui-${location.hostname}.crt`;
-
   return (
-    <CustomLayout
-      mainClassName="certificate-generate"
-      header={
-        <div className="p-panel__header is-sticky">
-          <h1 className="p-panel__title">
-            <HelpLink
-              href="https://github.com/canonical/lxd-ui/wiki/Authentication-Setup-FAQ"
-              title="Authentication Setup FAQ"
+    <CustomLayout mainClassName="certificates">
+      <Row>
+        <Col size={1} />
+        <Col size={10}>
+          <Notification
+            actions={[
+              ...(hasCertificate
+                ? []
+                : [
+                    {
+                      label: "I already have a client certificate",
+                      onClick: () => {
+                        navigate("/ui/login/certificate-add");
+                      },
+                    },
+                  ]),
+            ]}
```

Comment:
```suggestion
            actions={hasCertificate
                ? []
                : [
                    {
                      label: "I already have a client certificate",
                      onClick: () => {
                        navigate("/ui/login/certificate-add");
                      },
                    },
                  ]}
```
We can remove one level of nesting here.